tests/gc/gcleak3.nim

```diff
@@ -18,7 +18,8 @@ for i in 0..1024:
 
 proc limit*[t](a: var seq[t]) =
   var loop = s.len() - 512
-  for i in 0..loop:
+  for i in 0..<s.len:
+    if i > high(s): break
```

Comment:
Redundant check, we know that i < s.len inside the loop body.

Reply:
In line 26, the sequence that is iterated on gets shortened by `s.delete()`. 
I've changed the loop to `for i in high(int)` to clarify the code.

Reply:
done